composer.json

```diff
@@ -20,11 +20,12 @@
   },
   "minimum-stability": "stable",
   "require": {
-    "php": ">=7.0",
-    "rexlabs/utility-belt": "^3.0"
+    "php" : "^8.0",
+    "rexlabs/utility-belt": "dev-feat/github-tests",
+    "dms/phpunit-arraysubset-asserts": "^0.5.0"
```

Comment:
Should this package be included under `require-dev` if it's only needed to support the tests?

Reply:
Yes

Reply:
Updated